core/src/main/java/com/web3auth/core/Web3Auth.kt

```diff
@@ -182,10 +182,11 @@ class Web3Auth(web3AuthOptions: Web3AuthOptions) {
                     this.authorizeSession().whenComplete { resp, error ->
                         if (error == null) {
                             web3AuthResponse = resp
-                            initializeCf.complete(null)
                         } else {
                             print(error)
+                            initializeCf.completeExceptionally(error)
```

Comment:
We shouldn't return the error. Session not found is normal scenario, and if we return the error. `initialize` function will start throwing the error for fresh installs as well.

Reply:
Before 8.0.0 as well we didn't throw error. https://github.com/Web3Auth/web3auth-android-sdk/blob/362b004b72921e4d94a01c1b9d23532f54aee9b1/core/src/main/java/com/web3auth/core/Web3Auth.kt#L169

Reply:
done.